src/events/index.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash');
+
+const eventType = require('./types');
+
+function resolveHandler(event) {
+  // eslint-disable-next-line
+  const handler = require(`./handlers/${event}`);
+
+  if (!_.isFunction(handler)) {
+    throw new Error(`Invalid event handler resolved for event ${event}. Expected function`);
+  }
+
+  return handler;
+}
+
+/**
+ *  Sometimes it might be useful to ignore particular events for some reasons(debugging, deprecation and etc.)
+ *  Put event name here so no handler would be registered.
+ */
+const temporaryDisabledTypes = [];
+const eventTypes = _.without(_.values(eventType), ...temporaryDisabledTypes);
```

Comment:
probably it's better to define handlers explicitly. then we will not need these disabled types. But I don't mind this approach